systemFlake.nix

```diff
@@ -66,14 +67,26 @@ let
       extraArgs = sharedExtraArgs // lhs.extraArgs // rhs.extraArgs;
     };
 
+  foldHosts = 
+    let 
+      mergeOutput = lhs: rhs:
```

Comment:
I am fine with it - but i doubt that there is any need to move out this `mergeOutput` function to `let` statement

Reply:
good point. I did it originally because of the `getOutput` function, which is no longer there.